src/reaktoro_pse/reaktoro_block.py

```diff
@@ -0,0 +1,952 @@
+            domain=IsInstance((VarData, Var, dict, IndexedVar)),
+            description="Input pressure for reaktoro block",
+            doc="Var or IndexedVar that references system pressure",
+        ),
+    )
+    CONFIG.declare(
+        "pressure_indexed",
+        ConfigValue(
+            default=True,
+            domain=bool,
+            description="Pressure is indexed",
+            doc="""Option that defines how to treat input variable when building indexed reaktoroBlock":
+                - If true, the input has same indexing as block, and each indexed input will be passed into respective indexed reaktoroBlock
+                - If false, all indexed blocks will get same input""",
+        ),
+    )
+    CONFIG.declare(
+        "pH",
+        ConfigValue(
+            default=None,
+            domain=IsInstance((VarData, Var, dict, IndexedVar)),
+            description="Input pH for reaktoro block",
+            doc="Var or IndexedVar that references system pH",
+        ),
+    )
+    CONFIG.declare(
+        "pH_indexed",
+        ConfigValue(
+            default=True,
+            domain=bool,
+            description="pH is indexed",
+            doc="""Option that defines how to treat input variable when building indexed reaktoroBlock":
+                - If true, the input has same indexing as block, and each indexed input will be passed into respective indexed reaktoroBlock
+                - If false, all indexed blocks will get same input""",
+        ),
+    )
+    CONFIG.declare(
+        "aqueous_solvent_specie",
+        ConfigValue(
+            default="H2O",
+            domain=str,
+            description="Defines aqueous specie to use when speciating system",
+            doc="""When speciating, the H2O is fixed, while H and O is unfixed to allow system to equilibrate,
+            this should be same specie as being passed in composition, it will be automatically translated to reaktoro notation
+            if enabled""",
+        ),
+    )
+    CONFIG.declare(
+        "exact_speciation",
+        ConfigValue(
+            default=False,
+            domain=bool,
+            description="Defines if exact speciation is provided ",
+            doc="If true, then the problem will constrain all elements to equal to provided inputs after dissolution",
+        ),
+    )
+    CONFIG.declare(
+        "mineral_phases",
+        ConfigValue(
+            default=None,
+            description="List or str of mineral phases",
+            doc="""
+            Mineral phases supported by selected data base
+            Accepts:
+                string name for phase
+                reaktoro.MineralPhase initialized object (e.g reaktoro.MineralPhase('Calcite'))
+                List of strings or reaktor.MineralPhases
+            """,
+        ),
+    )
+    CONFIG.declare(
+        "gas_phases",
+        ConfigValue(
+            default=None,
+            description="List or str for gas phases",
```

Comment:
hmm - I think I missnamed this now that I think about it. There is only one gas phase in reaktoro - but I refer to gas phases as also gasess -I will update the naming convention to be more clear here.

Reply:
To be more clear - there is one "gas phase object" that contains multiple gasses - so I will change it to gas_phase and clarify that it can contain multiple gas species.